### DISCUSSION TITLE
Calibrate Phidget attached Thermometer - Artisan BT Reading Too Low

### DISCUSSION THREAD

**Discussion by gevans3** [artisan-roaster-scope/artisan#988] [Q&A]
Tried to make sense of my issue from this discussion and I'm getting close.  My problem, not being a programmer but a home roaster, is that based on feedback I'm getting from others in my roasting forums and from various available roast profiles for my Hot Top Roaster, it appears that my roaster BT is consistently reading out in Artisan about 20 degrees too low.  In other words, if I'm expecting a first crack at 175, I'm getting it around 155.  This 20 degree difference is pretty consistent along the curve.  If a roast profile tells me to charge at 150, I suspect I'm actually charging at 130.  Based on a suggestion I received from Ingo at Roast Rebels, I'm going to run a manual roast sas Artisan to see if, perhaps, the thermocouples are the issue and not the way they're translating to Artisan (taking Artisan out of the equation to eliminate that as a variable).  Barring that, I'd like to know what formula to put in to the BT offset (x+20?) to index my BT redoubts in Artisan Scope. 

If any of this makes sense, any insight would be appreciated.  Thank you.

### Discussed in https://github.com/artisan-roaster-scope/artisan/discussions/809

<div type='discussions-op-text'>

<sup>Originally posted by **techmaster70** February  3, 2022</sup>
I have a Phidget 1048-0 and 2 Thermometers (thermocouples) attached. I can see they are off by 4-8 degrees F, but I can't figure how to 'adjust' or calibrate them. Phidget tells me it's via the software displaying the temp (so, Artisan) and send me some Python code examples.
Can anyone help with this?
I'm slightly decent at Ansible, and been exposed to Python, but really would rather find an interface setting to adjust via gui since I'm not a programmer and really have no idea where to start here. (I searched this forum but found nothing about this).

TIA
I have an Artisan 3-e roaster and Mac PowerBook Pro with OS 10.15.7.</div>

**Comment by MAKOMO** [comment]
<img width="589" alt="Screenshot 2022-10-20 at 08 19 47" src="https://user-images.githubusercontent.com/13909011/196871111-d738dfb0-2987-4e86-9afb-3b02db6bc8a7.png">

**Reply by gevans3** [reply]
Thank you. Here is an example (this one, obviously, is calibrated as Fahrenheit, but I get the idea.  Just wanted to illustrate what I was talking about re: input I’m receiving that my BT is indicating low.  So appreciative.

![6FE145ED-8C93-4D12-B8A9-DB6C8B49650A](https://user-images.githubusercontent.com/7534593/196926745-b4d717fa-aa3d-446a-9bdc-7b855466a204.png)
